schema/deploy/tables/cif_attachment.sql

```diff
@@ -0,0 +1,42 @@
+-- Deploy cif:tables/cif_attachment to pg
+
+BEGIN;
+
+create table cif.cif_attachment
```

Comment:
```suggestion
create table cif.attachment
```
I don't know if we need the `cif_` prefix on tables (other than `cif_user`), since it's already under a `cif` schema @pbastia @dleard what do we think? We'll want to be consistent here

Reply:
I think it makes sense to simplify. I have updated the table to be just attachment.